main/src/library/Int16.flix

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2016 Liam Palmer
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Int16 {
+
+    /**
+     * Returns the number of bits used to represent an `Int16`.
+     */
+    def size: Int32 = 16
+
+    /**
+     * Returns frequently used Int16 values for readability.
+     */
+    def zero: Int16 = 0i16
```

Comment:
I would prefer to avoid these. I think it is fine to have just `0i16`. That is the way to write zero as a 16-bit integer in Flix.